src/eba_handlers.eliom

```diff
@@ -94,6 +94,12 @@ let send_act email userid =
   Lwt.return ()
 
 let sign_up_handler' () email =
+  let send_act email userid =
+    let msg =
+      String.concat ", " ["Welcome!\r\nTo confirm your e-mail address";
+                          "please click on this link: "] in
```

Comment:
I suspect that you had a space after `\`; if you remove that, the whitespace on the next line (before "please") will get ignored :).